### ISSUE TITLE
Neovim default status line flashes at start

### ISSUE THREAD

**Issue by gbaranski** [glepnir/galaxyline.nvim#162]
Neovim default status line flashes at start, is there any way to prevent that? Default status line is visible for few milliseconds at start. 

https://user-images.githubusercontent.com/34353640/117567344-384f6680-b0bc-11eb-9e79-12fb39967557.mp4

**Comment by WhyNotHugo** [comment]
Hmm... interesting. Mine flashes which, and I though it might be related to my theme or something funky on my end.

I'm curious what the highlight group is for that space _before_ galaxyline render. Merely setting that to the default background would make the flashing almost unnoticeable.

**Comment by kdav5758** [comment]
Mine flashes too... thought I was the only one.

**Comment by ram02z** [comment]
I only notice the flashing on my slower systems.

**Comment by peterwu** [comment]
Same problem here.

**Comment by arpangreat** [comment]
Did you previously install and use airline on your nvim ??

**Comment by gbaranski** [comment]
> Did you previously install and use airline on your nvim ??

Yes but that was on previous system installation, and before whole config rewrite.

**Comment by arpangreat** [comment]
> > Did you previously install and use airline on your nvim ??
> 
> Yes but that was on previous system installation, and before whole config rewrite.

Okay that's great ...
will you set this on your config and see if this helps??
```vim
let g:airline_disable_statusline = 1
```

**Comment by gbaranski** [comment]
> let g:airline_disable_statusline = 1

Nope, it don't change anything.

**Comment by WhyNotHugo** [comment]
Never had airline here.

**Comment by mortezadadgar** [comment]
it's noticeable on large repositories even without git branch section [config](https://github.com/mortezadadgar/dotfiles/blob/master/nvim/.config/nvim/lua/statusline.lua)

**Comment by bluretrece** [comment]
Does anyone were able to solve this isssue? Mines flashes as well but it last longer than yours, around ~1.5 seconds. I started noticing this after updating the plugin and building the latest neovim version. I though that with the following code `gl.load_galaxyline()` at the bottom would make sure to load galaxyline's components beforehand to avoid lag issues at startup, but overall the plugin it's noticeable slower now.

**Comment by peterwu** [comment]
I seem to have found a workaround:

in galaxyline.vim file, put in this line to reset the StatusLine highlight to NONE.

`highlight StatusLine ctermfg=NONE ctermbg=NONE guifg=NONE guibg=NONE`

so that, it looks like this:


```
let g:loaded_galaxyline = 1

highlight StatusLine ctermfg=NONE ctermbg=NONE guifg=NONE guibg=NONE
lua require('galaxyline').galaxyline_augroup()
```

I'll open a PR for this.

**Comment by mks1508** [comment]
That'll make you less painful
```lua
-- make sure this line loaded before galaxyline config
vim.api.nvim_command("hi StatusLine guibg=<your_background_color> guifg=<your_background_color>")

<galaxyline config here>
```

**Comment by Pocco81** [comment]
@mks1508 better than nothing at least!

**Comment by NTBBloodbath** [comment]
Hey so I forked galaxyline very recently, added a few goodies and merged some upstream PRs. This issue is gone for me, since I removed some pseudo-async code and stopped lazy-loading galaxyline.

About startup time, it is taking less than 0.5ms to load to me, I guess I optimized it a bit much haha, feel free to [give it a try](https://github.com/NTBBloodbath/galaxyline.nvim).
